content/source/docs/enterprise/saml/identity-provider-configuration-aad.html.md

```diff
@@ -0,0 +1,68 @@
+sidebar_current: "docs-enterprise2-private-saml-identity-provider-configuration-aad"
+---
+
+# AAD Configuration
+
+Follow these steps to configure Azure Active Directory (AAD) as the identity provider (IdP) for Private Terraform Enterprise (PTFE).
+
+This guide assumes you have an appropriate licensing agreement for Azure Active Directory that supports non-gallery appplication single sign-on.
+
+## Configure a New AAD Non-Gallery Application
+
+1. In the Azure portal, navigate to Azure Active Directy > Enterprise Applications and select Add an Application.
+  ![Screenshot: AAD's "Add Application" page](./images/sso-aad-saml-add-application.png)
+2. Select Non-gallery application. Provide a name for the application and click Add.
+  ![Screenshot: AAD's dialog for beginning a new application](./images/sso-aad-saml-new-application-form)
+3. AAD should automatically redirect to your new application settings, navigate to Single sign-on and select SAML
+  ![Screenshot: the first page of AAD's new app configuration workflow, called "General Settings"](./images/sso-aad-saml-sso-method.png)
+4. Select the pencil within the Basic SAML Configuration and configure the following settings with the specified values:
+    - **Identifier (Entity ID):** `https://<TFE HOSTNAME>/users/saml/metadata` (listed as "Metadata (audience) URL" in TFE's SAML settings).
+    - **Reply URL (Assertion Consumer Service URL):** `https://<TFE HOSTNAME>/users/saml/auth` (listed as "ACS consumer (recipient) URL" in TFE's SAML settings).
+    - **Sign on URL:** `https://<TFE HOSTNAME>/`
+    ![Screenshot: The "Configure SAML" page of ADD's new app workflow, with the specified settings entered.](./images/sso-aad-saml-configuration.png)
+5. In the User Attributes & Claims section select the pencil and configure the following items:
+    - **Name Identifier value:** `user.mail`
+    ![Screenshot: Modifying the name identifier value.](./images/sso-aad-saml-user-claims-name-identifier.png)
+6. Still in the "User Attributes & Claims" page, configure a user claim to map which team a user belongs to. Under "Manage user claims" page, configure the following:
+    - **Name:** `MemberOf` (This is the default name for TFE's group [attribute][./attributes.html]; the name of this attribute can be changed in [TFE's SAML settings](./configuration.html) if necessary.)
+    - **Source attribute:** (drop-down): user.assignedroles (Custom roles will be created in Azure Active Directory that will be used to map users and groups to TFE teams.)
+    ![Screenshot: Add MemberOf claim](./images/sso-aad-saml-user-claims-memberof.png)
+7. Under the "SAML Signing Certificate" header, download the signing certificate in base64 format
+    ![Screenshot: Download the SAML signing certificate](./images/sso-aad-saml-signing-certificate.png)
+8. Still in the "Single Sign On Configuration" page, under the "Set up <AAD App Name>", copy the following urls that we will enter in our TFE configuration to link TFE to AAD:
```

Comment:
```suggestion
8. Still in the "Single Sign On Configuration" page, under "Set up <AAD App Name>", copy the following URLs that will be provided to TFE to link TFE to AAD:
```
"Still" caught me off guard here, because last I checked we were somewhere else (User Attributes & Claims). Can you maybe clarify this language a bit?

Reply:
Tried to clarify this by referring to the header we should be looking at on the page. The page is the Single Sign On Config page however it may not be clear to users as they will just see the headers and different sections